src/docs/goals/platforms.md

```diff
@@ -3,18 +3,44 @@ title: "Platform"
+
+Our goals through AuxEng provide benefits to any team developing or maintaining a platform.
+
+## Product, Platform, AuxEng
+
+In [lean software development](http://theleanstartup.com/principles), a software team builds code that might solve a problem. The team then measures the result through usage analytics or data on the affect their code may have on their users. The team _learns_ from those measurements and synthesizes ideas on what to build next.
+
+We believe in this methodology. A goal of any engagement in AuxEng is to provide measurement and learning during engagements, then fresh ideas and methodology for building off-engagement.
+
+### During Engagements
+
+When an engineer works with another team, they are pair programming with the intended users of a platform. Part of that experience includes seeing where their products fall short or don't solve a problem effectively. If it takes 20 minutes to learn how to use a tool that saves 20 minutes, people will likely not use that tool. An Aux Engineer can learn this during an engagement; then learn the ways that their users _expect_ software or platforms to work (this is a less-data-driven approach to _measuring_ the success of any built changes). The Aux Engineer can relay feedback and proposed changes during or after the engagement (sharing _learning_) to appropriately serve real users engaging with a platform.
+
+The contributions an Aux Engineer can glean are hugely valuable. Keep in mind: they're doing this not only to serve a platform team's product and roadmap. Aux Eng actively moves the needs of the business forward, faster, by connecting experts in a domain/platform with others that need help engaging in that space. In our experience, that exposure from our team into our user teams is as valuable as building a critical change into our platform.
+
+### Off-Engagements
+
+As we've served our goals effectively on-engagement, it's important to highlight the goals off-engagement. The perspective that an Aux Engineer brings back to a team during an engagement is important. The perspective they can bring when they're not pairing 4 days a week with another team is just as vital to team success and productivity.
+
+It's a goal of Aux Eng programs to bring perspective and understanding of the software landscape and engineering culture back to a platform team. After working with teams in one or many spaces separate from the platform team, an engineer will promote difficulty that real teams have using products and processes back to a platform team. Our goal is to ultimately influence a product roadmap or design choices in a system toward the needs of actual users.
```

Comment:
Wonder if a different word than `promote` makes more sense here.. convey?

Reply:
I like `convey`, or something around "communicate the difficulties that real teams have"